gpu_support/test_cuda

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Test CUDA
+module use /cvmfs/pilot.eessi-hpc.org/host_injections/nvidia/modules/all/
+module load CUDA
+tmp_dir=$(mktemp -d)
+cp -r $EBROOTCUDA/samples $tmp_dir
```

Comment:
CUDA no longer ships samples, there is a plan to ship the (compiled) CUDA samples with EESSI